pkg/operation/botanist/waiter.go

```diff
@@ -82,16 +77,20 @@ func (b *Botanist) WaitUntilTunnelConnectionExists(ctx context.Context) error {
 			eventsErrorMessage, err2 := kutil.FetchEventMessages(ctx, b.K8sShootClient.Client().Scheme(), b.K8sShootClient.Client(), service, corev1.EventTypeWarning, 2)
 			if err2 != nil {
 				b.Logger.Errorf("error %v occurred while fetching events for VPN load balancer service", err2)
-				return retry.SevereError(fmt.Errorf("'%w' occurred but could not fetch events for more information", err))
+				return fmt.Errorf("'%w' occurred but could not fetch events for more information", err)
 			}
 
 			if eventsErrorMessage != "" {
-				return retry.SevereError(fmt.Errorf("%s\n\n%s", err.Error(), eventsErrorMessage))
```

Comment:
Wasn't it enough to just change this to `retry.MinorError`?

Reply:
If we change to `retry.MinorError`, then will fetch the Service events at each retry until the timeout exceed. Why would we want to do this when we can fetch the events only once after the timeout is exceeded?

Reply:
I see, thanks for explaining.